test_args.sh

```diff
@@ -217,7 +217,7 @@ run_lp -broadcaster -verifierUrl http://host -verifierPath path
 kill $pid
 
 # Check OK with verifier + external storage
-run_lp -broadcaster -verifierUrl http://host -s3bucket foo/bar -s3creds baz/bat
+run_lp -broadcaster -verifierUrl http://host
```

Comment:
Just stripping the last 2 flags without replacing them changes the behavior of this test. The test was originally meant to check that the node starts if -verifierUrl is specified *and* an object store is configured. Since the -s3bucket and -s3creds flags are being removed we could replace them with -objectStore which can be used to accomplish the same configuration. See https://discord.com/channels/423160867534929930/426106677650128898/829774858979246080. 

Ex. `-objectStore s3+https://baz:bat@s3api.example.com/foo/bar`

Reply:
I think this should be fixed in the latest commit